packages/core/src/App/Containers/Layout/header/header.tsx

```diff
@@ -42,11 +43,6 @@ const TradersHubHeaderWallets = makeLazyLoader(
     () => <HeaderFallback />
 )();
 
-const DTraderV2ContractDetailsHeader = makeLazyLoader(
```

Comment:
The issue is that styles was overwritten because of the loading order of css. When switching to Contract Details, we lazy-load component with quill styles (in Network a new chunk is added). As they arrive later, they overwrite some original css properties.